HacktoberfestProject.Web/Tools/Guard.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace HacktoberfestProject.Web.Tools
+{
+    public static class Guard

```

Comment:
Could we rename this to something more descriptive such as "NullChecker"

Reply:
Done!
FYI: the name was based on [`Guard Clause`](http://c2.com/cgi/wiki?GuardClause)